Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.17
+
+RUN apt-get update && apt-get -y dist-upgrade
+RUN apt-get -y install build-essential libssl-dev libffi-dev libblas3 libc6 liblapack3 gcc python3-dev python3-pip cython3
+RUN apt-get -y install python3-numpy python3-scipy 
+RUN apt install -y netcat
```

Comment:
I don't think netcat is required here.

Reply:
netcat is required to run this command from terminal inside docker container
```(printf '*1\r\n$4\r\nPING\r\n*3\r\n$3\r\nSET\r\n$1\r\nk\r\n$1\r\nv\r\n*2\r\n$3\r\nGET\r\n$1\r\nk\r\n';) | nc localhost 7379```

Reply:
sure but you can also run this from outside of the container also. Is this for testing purpose?